core/src/main/java/org/frankframework/monitoring/Trigger.java

```diff
@@ -66,20 +64,57 @@ public class Trigger implements ITrigger {
 	private Queue<Instant> eventDates = null;
 	private boolean configured = false;
 
+	/**
+	 * Default Trigger constructor
+	 */
+	public Trigger() {
+		// No op
+	}
+
+	/**
+	 * Copy-constructor for trigger used to validate updates before applying them.
+	 *
+	 * @param trigger The {@link ITrigger} to be copied.
+	 */
+	public Trigger(ITrigger trigger) {
+		this.severity = trigger.getSeverity();
+		this.sourceFiltering = trigger.getSourceFiltering();
+		this.triggerType = trigger.getTriggerType();
+		this.eventCodes.addAll(trigger.getEventCodes());
+		this.threshold = trigger.getThreshold();
+		this.period = trigger.getPeriod();
+		if (trigger.getAdapterFilters() != null) {
+			this.adapterFilters.putAll(trigger.getAdapterFilters());
+		}
+	}
+
+	@Override
+	public void validate() throws ValidationException {
```

Comment:
Dit moet allemaal in configure gebeuren, een losse methode voegt niets toe en voegt alleen maar verwarring en complexiteit toe.